scalanet/src/io/iohk/scalanet/peergroup/kademlia/KRouter.scala

```diff
@@ -318,6 +387,52 @@ class KRouter[A](
 object KRouter {
   case class Config[A](nodeRecord: NodeRecord[A], knownPeers: Set[NodeRecord[A]], alpha: Int = 3, k: Int = 20)
 
+  /**
+    * Enrolls to kademlia network from provided bootstrap nodes and then start handling incoming requests.
+    * Use when finishing of enrollment is required before starting handling of incoming requests
+    *
+    * @param config discovery konfig
```

Comment:
I think it would be funny to replace all instances of the letter 'c' with the letter 'k' in these kademlia packages.